e2e-tests/batch-mode-tests.sh

```diff
@@ -150,19 +161,33 @@ function test_fhir_sink() {
   "${command[@]}"
 
   cd "${test_dir}"
-  mkdir omrs_fhir_sink
   print_message "Finding number of patients, encounters and obs in openmrs server"
-  curl -L -X GET -u admin:Admin123 --connect-timeout 5 --max-time 20 \
-    http://localhost:8099/openmrs/ws/fhir2/R4/Patient/ 2>/dev/null >>./omrs_fhir_sink/patients.json
-  total_test_patients=$(jq '.total' ./omrs_fhir_sink/patients.json)
+  openmrs_query omrs_fhir_sink
 
   mkdir fhir
-  print_message "Counting number of patients, encounters and obs sinked to fhir files"
   curl -L -X GET -u hapi:hapi --connect-timeout 5 --max-time 20 \
     http://localhost:8098/fhir/Patient/?_summary=count 2>/dev/null >>./fhir/patients.json
+
+  curl -L -X GET -u hapi:hapi --connect-timeout 5 --max-time 20 \
+    http://localhost:8098/fhir/Encounter/?_summary=count 2>/dev/null >>./fhir/encounters.json
+
+  curl -L -X GET -u hapi:hapi --connect-timeout 5 --max-time 20 \
+    http://localhost:8098/fhir/Observation/?_summary=count 2>/dev/null >>./fhir/obs.json
+
+
+  print_message "Counting number of patients, encounters and obs sinked to fhir files"
+
   total_patients_sinked_fhir=$(jq '.total' ./fhir/patients.json)
   print_message "Total patients sinked to fhir ---> ${total_patients_sinked_fhir}"
-  if [[ ${total_patients_sinked_fhir} == ${total_test_patients} ]]; then
+
+  total_encounters_sinked_fhir=$(jq '.total' ./fhir/encounters.json)
+  print_message "Total encounters sinked to fhir ---> ${total_encounters_sinked_fhir}"
+
+  total_obs_sinked_fhir=$(jq '.total' ./fhir/obs.json)
+  print_message "Total observations sinked to fhir ---> ${total_obs_sinked_fhir}"
+
+  if [[ ${total_patients_sinked_fhir} == ${total_test_patients} && ${total_encounters_sinked_fhir} == ${total_test_encounters} && ${total_obs_sinked_fhir} == ${total_test_obs} ]] \
```

Comment:
nit: Please try to break long lines. The [style guide](https://google.github.io/styleguide/shellguide.html#line-length-and-long-strings) says 80 characters; we don't need to be pedantic about it (use it as a guide) but I would say let's never go beyond 100.